pkg/controllers/management/node/controller.go

```diff
@@ -46,9 +46,10 @@ import (
 )
 
 const (
-	defaultEngineInstallURL       = "https://releases.rancher.com/install-docker/17.03.2.sh"
-	amazonec2                     = "amazonec2"
-	userNodeRemoveFinalizerPrefix = "clusterscoped.controller.cattle.io/user-node-remove_"
+	defaultEngineInstallURL         = "https://releases.rancher.com/install-docker/17.03.2.sh"
```

Comment:
The default version of docker being used for RKE is now 20.10. Not sure if we need to bump this in 2.6 but if we need to I'll backport my PR https://github.com/rancher/rancher/pull/39318.